### PR TITLE
Only track training loss, use ml.p2.xlarge in the example notebook, hold reference to TrainingJobAnalytics object.

### PR DESCRIPTION
**Pull request by ksaaskil**
- I assumed that `sagemaker.analytics.TrainingJobAnalytics(training_job_name=...)` would return a monotonically growing dataframe that would have metrics in a monotonously increasing order. This is not true: at any step new rows can be prepended to the existing ones with earlier timestamps. Not sure how this all works but the current example does not work. I'll try and fix that soon but this fixes the example.
- Hold a reference to TrainingJobAnalytics by job name to avoid re-creating it every time.

### CODE REVIEW

File: examples/sagemaker/pytorch_rnn_meeshkan.ipynb [hunk 1]
```diff
@@ -282,15 +280,13 @@
     "                    role=role,\n",
     "                    framework_version='1.0.0',\n",
     "                    train_instance_count=1,\n",
-    "                    train_instance_type='ml.m4.xlarge',  # Use e.g. `ml.p2.xlarge` to train faster\n",
+    "                    train_instance_type='ml.p2.xlarge',\n",
```
Comment (idantene) [inline examples/sagemaker/pytorch_rnn_meeshkan.ipynb]: Maybe keep the same `ml.m4.xlarge` with that comment

File: meeshkan/core/sagemaker_monitor.py [hunk 2]
```diff
@@ -253,6 +256,7 @@ def get_new_records(df_new: pd.DataFrame, df_old: Optional[pd.DataFrame] = None)
                         None,
                         self.sagemaker_helper.get_training_job_analytics_df, job.name)
                     if not metrics_df.empty:
+                        metrics_df = metrics_df.sort_values(by="timestamp")
```
Comment (idantene) [inline meeshkan/core/sagemaker_monitor.py]: Let's add a `TODO` note to filter by timestamp-value name, as we do in `TrackerBase`.

**Comment by ksaaskil** [comment]
Removed the sorting as it breaks the indexing [here](https://github.com/Meeshkan/meeshkan-client/blob/dev/meeshkan/core/sagemaker_monitor.py#L221).
